rasa/train.py

```diff
@@ -77,12 +77,8 @@ def train(
     Returns:
         Path of the trained model archive.
     """
-    config = get_valid_config(config, CONFIG_MANDATORY_KEYS)
-
+    config = _get_valid_config(config, CONFIG_MANDATORY_KEYS)
     train_path = tempfile.mkdtemp()
-    old_model = model.get_latest_model(output_path)
-    retrain_core = True
-    retrain_nlu = True
 
     skill_imports = SkillSelector.load(config)
```

Comment:
Should we move lines 80 to 95 to a function `_get_training_data`? Or is that a overkill?

Reply:
As it is not just getting training data but many different kind of things that are needed later on, I would keep it as it is.